main.go

```diff
@@ -151,5 +157,13 @@ func run(c *cli.Context) error {
 		},
 	}
 
+	for _, s := range c.StringSlice("git-configs") {
```

Comment:
IMHO you don't even need to split these strings, just keep the slice of strings and iterate through that with `git config --global STRING`

Reply:
I tried at my Linux box, it didn't work.
![image](https://user-images.githubusercontent.com/2606021/62367201-7c165300-b55b-11e9-8e08-abe2ec769f87.png)

Reply:
You are right, it must be supplied as to attributes.